test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -706,55 +707,89 @@ def __are_requirements_matching(param_dic=None, test=None):
             param_count['num_' + key] = len(param_dic[key])
         for pk in param_count:
             if param_count[pk] < eff_tc_req[pk]:
-                return False
+                _msg = 'available ' + pk + " is less than requirement"
```

Comment:
less than "required"
also log what is the required amount.

Reply:
@prakashcv13  it looks like "available num_servers is less than requirement".
there is one skipped reason in atteched log file "changed_default_requirement_and_skip_msg.txt"

Reply:
tester can get idea, it is comparing with number available and requirement.

Reply:
i reviewed the log file. I am suggesting that we display both available and required number in the log, something like - 
available num_moms (1) is less than required (2). 1 and 2 is used just as an example.

Reply:
We are comparing with key num_moms/num_servers/num_comms/num_clients that is defined in default_requirement dictionary. default dictionary is used at so many places, so it will confuse at other places. So i think it is ok to use, as it is defined in defualt_requirements. 
default_requirements = {
    'num_servers': 1,
    'num_moms': 1,
    'num_comms': 1,
    'num_clients': 1,